### ISSUE TITLE
Mute / block by owner

### ISSUE THREAD

**Issue by RichGriese** [tlkio/feedback#5]
At some point I believe it will be important to allow owners to mute and/or block people. This might apply only to communities that are set up as non-anons.

**Comment by antstorm** [comment]
@RichGriese yes, this is definitely a feature a lot of users are waiting for. We've also got means in place to do this manually. The problem I see here is communicating this to tlk.io users without making it really confusing.

This would means that you'd need to become a moderator (for which you need to be logging with with TW or FB) and disable anonymous logins as well as. The flow here is not ideal to say the least.

Potentially the solution might be in giving up anonymous users and replacing them with a passwordless email login — you enter your email, we send you a confirmation link, which you click and then are able to choose a nickname. Your email won't be exposed to other users, however we can use it to prove your identity and control access.

**Comment by RichGriese** [comment]
@antstorm I would assume the feature would only work for moderators. Yeah... I look at tlk as a tool for non-anons. I am including no consideration for those using the tool in anon mode.

I would vote AGAINST email login. My interest in the tool is as an extension of Twitter. So, Twitter is responsible for all identification. I'm after a extension of the Twitter community. Where anyone posting in my tlk community could talk outside tlk simply via Tweets. If there was a email login possibility I would lobby for it to be implemented AFTER the implementation of checkbox for services I also recommend. So that... even if you allow a email login, in my community I can disable that (as I hope one day to be disable the FB login).  Cheers!

**Comment by antstorm** [comment]
@RichGriese that makes perfect sense and I totally get why you're pushing for a granular control over identity providers. Also note that 60% of our logged-in users are coming from Facebook, so we really need to account for them when making decisions here :)

As for the topic of this ticket — I would say that an email login is a pre-requisite to proper muting/blocking.
